<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome to Airport Information Generator</string>
    </property>
   </widget>
   <widget class="QLabel" name="summaryLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search using an airport code to see airport information</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="instructionText">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt;&quot;&gt;Instruction:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt;&quot;&gt;1. Select an airport code from dropdown.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt;&quot;&gt;2. Click &amp;quot;Search&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevResultButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Previous Search Result</string>
    </property>
   </widget>
   <widget class="QComboBox" name="airportSelect">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="airportSelectLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select an airport code:</string>
    </property>
   </widget>
   <widget class="QLabel" name="airportCodeLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>791</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Airport Code:</string>
    </property>
   </widget>
   <widget class="QLabel" name="airportNameLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>791</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Airport Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="locationLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>310</y>
      <width>791</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Location:</string>
    </property>
   </widget>
   <widget class="QLabel" name="covidLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>791</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>COVID Testing:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nearbyButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Show Nearby Restaurant(s)</string>
    </property>
   </widget>
   <widget class="QListWidget" name="nearbyList">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>391</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>978</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
